packages/sync-actions/src/customer-actions.js

```diff
@@ -126,3 +141,61 @@ export function actionsMapShippingAddresses(diff, oldObj, newObj) {
 
   return handler(diff, oldObj, newObj)
 }
+
+export function actionsMapAuthenticationModes(diff, oldObj, newObj) {
+  // eslint-disable-next-line no-use-before-define
```

Comment:
i though it was better to keep `buildAuthenticationModeActions` at the bottom and all the actionMaps together.